pom.xml

```diff
@@ -30,12 +29,44 @@
             <groupId>at.researchstudio.sat</groupId>
             <artifactId>won-bot</artifactId>
             <version>0.4</version>
+            <exclusions>
```

Comment:
lets put a pin in this for now, idk if this is really best practice if we exclude a dependency of a project that we maintain ourselves, maybe there is a more elegant way to figure this out